packages/web/docs/src/content/product-updates/2024-10-11-laboratory-improvements/index.mdx

```diff
@@ -6,9 +6,9 @@ date: 2024-10-11
 authors: [dimitri]
 ---
 
-import fullScreenMode from './full-screen-mode.mp4'
-import queryBuilder from './query-builder.mp4'
-import newTabs from './tabs-new.mp4'
+export const fullScreenMode = `${process.env.NEXT_BASE_PATH}/product-updates/full-screen-mode.mp4`
+export const queryBuilder = `${process.env.NEXT_BASE_PATH}/product-updates/query-builder.mp4`
+export const newTabs = `${process.env.NEXT_BASE_PATH}/product-updates/tabs-new.mp4`
```

Comment:
```suggestion
import { addBasePath } from 'next/dist/client/add-base-path'

export const fullScreenMode = '/product-updates/full-screen-mode.mp4'
export const queryBuilder = '/product-updates/query-builder.mp4'
export const newTabs = '/product-updates/tabs-new.mp4'
```

and `<source src={addBasePath(src)} type="video/mp4" />`

Reply:
Okay, so you mean we have `addBasePath` in `mdx-components` in `@theguild/components`? Fair dos.